src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.hpp

```diff
@@ -282,6 +287,7 @@ class SolidMechanicsLagrangianFEM : public SolverBase
   integer m_strainTheory;
   string m_contactRelationName;
   MPI_iCommData m_iComm;
+  integer m_effectiveStressFlag;
```

Comment:
Maybe a more meaningful attribute name for this variable, like `useFixedStressAssumptionInSolidMechanicsUpdate` (or something better :) )?

Reply:
> Maybe a more meaningful attribute name for this variable, like `useFixedStressAssumptionInSolidMechanicsUpdate` (or something better :) )?

Thanks. Yes, I have changed it to m_fixedStressUpdateThermoPoroElasticityFlag. I think it can make others clear about the coupling option and the physics considered in the solid mechanics update.